autocomplete/table.go

```diff
@@ -20,7 +19,7 @@ func GetTableAutoCompleteSuggestions(schema *schema.Metadata, connectionMap *ste
 	// schema names
 	var schemasToAdd []string
 	// unqualified table names - initialise to the reflection table names
-	unqualifiedTablesToAdd := constants.ReflectionTableNames()
+	unqualifiedTablesToAdd := []string{} // constants.ReflectionTableNames()
```

Comment:
remove commented out code. Do we still need constants.ReflectionTableNames()?

Reply:
We still need it. There are other places which refer to the `constants.ReflectionTableNames()` function.